lib/mongo/client.rb

```diff
@@ -266,6 +267,12 @@ def hash
     # @option options [ String ] :log_prefix A custom log prefix to use when
     #   logging. This option is experimental and subject to change in a future
     #   version of the driver.
+    # @option options [ Integer ] :max_connecting The maximum number of
+    #  connections that can be connecting simultaneously. The default is 2.
+    #  This option should be increased if there are many threads that share
+    #  the same client and the application is experiencing timeouts
+    #  while waiting for connections to be established.
+    # selecting a server for an operation. The default is 100.
```

Comment:
"... The default is 100"

Is that a copy/paste artifact?

Reply:
It is indeed, good catch! Thank you, fixed.